### DISCUSSION TITLE
インストール失敗

### DISCUSSION THREAD

**Discussion by 11EDA** [aclap-dev/video-downloadhelper#662] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://missav.com/dm2/ja/heyzo-0566",
  "error": {
    "details": "SideDownload error: : [in#0 @ 000001ca3ab7b840] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/05c21db4-1757-4bc6-89c7-aff2e1044a24/1280x720/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_1860204000124225",
    "error": "coapp_failure",
    "id": "coapp_failure_188e9a9a-3ce2-4a0b-8a98-04d0eb0634ce"
  }
}

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: ja
coapp: {"found":true,"new_version":false,"path":"D:\\Downloads\\DL\\DownloadHelper\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
Problems on the site missav have been reported before.  Read this: #519.  Be sure you click any links you see that say, "Show x previous replies."
